lab-bundle-spec.md

```diff
@@ -271,13 +269,90 @@ No additional attributes
   id: primary_domain
 ```
 
-##### Future Resource Types
+##### AWS Account (aws_account)
+
+attribute   | required | type       | notes
+----------- | -------- | ---------- | ----------------------------------------
+account_restrictions.allow_dedicated_instances  |          | boolean | Default false.
+account_restrictions.allow_spot_instances       |          | boolean | Default false.
+account_restrictions.allow_subnet_deletion      |          | boolean | Default false.
+account_restrictions.allow_vpc_deletion         |          | boolean | Default false.
+account_restrictions.allowed_ondemand_instances |          | array   | Array of [EC2 instance types](#valid-eC2-instance-types) that are valid for Ondemand usage. Default none.
+account_restrictions.allowed_rds_instances      |          | array   | Array of [EC2 instance types](#valid-eC2-instance-types) that are valid for RDS usage. Default none.
+startup_script.type              |          | string  | The type of startup script. Only `cloud_formation` is supported.
+startup_script.path              |          | path    | Relative path to a directory tree with the script contents.
+startup_script.custom_properties |          | array   | Array of pairs. See [Custom Script Properties](#custom-script-properties) for details.
+cleanup_script.type              |          | string  | The type of cleanup script. Only `cloud_formation` is supported.
+cleanup_script.path              |          | path    | Relative path to a directory tree with the script contents.
+cleanup_script.custom_properties |          | array   | Array of pairs. See [Custom Script Properties](#custom-script-properties) for details.
+user_policy                      |          | path    | Relative path to a [JSON policy](https://awspolicygen.s3.amazonaws.com/policygen.html) document.
+
+```yml
+- type: aws_account
+  id: the_account
+  variant: sts
+  startup_script:
+    type: cloud_formation
+    path: ./lab.template
+    custom_properties:
+      - key: userNameWindows
+        value: student
+  user_policy: ./student.policy
+  account_restrictions:
+    allow_dedicated_instances: false
+    allow_spot_instances: false
+    allow_subnet_deletion: false
+    allow_vpc_deletion: false
+    allowed_ondemand_instances: []
+    allowed_rds_instances: ['db.t2.micro']
+```
+
+###### Variants for AWS Account
+
+- aws_beta
+- aws_ec2
+- aws_rt53labs_ilt
+- aws_sts
+- aws_vpc
+- awsiam
```

Comment:
in the current method of launching aws accounts, 'awsiam' and 'awsfed' are separate fleet types. which of those should be aws_account, and are they similar enough that they should be considered variants or should they be different resource types? (cc @eniskonuk)